plugins/API/API.php

```diff
@@ -240,56 +227,6 @@ protected function getSegmentValuesFromVisitorLog($segmentName, $table)
         return $values;
     }
 
-    /**
-     * Returns the url to application logo (~280x110px)
-     *
-     * @param bool $pathOnly If true, returns path relative to doc root. Otherwise, returns a URL.
-     * @return string
-     * @deprecated since Matomo 2.15.1
-     */
-    public function getLogoUrl($pathOnly = false)
-    {
-        $logo = new CustomLogo();
-        return $logo->getLogoUrl($pathOnly);
-    }
-
-    /**
-     * Returns the url to header logo (~127x50px)
-     *
-     * @param bool $pathOnly If true, returns path relative to doc root. Otherwise, returns a URL.
-     * @return string
-     * @deprecated since Matomo 2.15.1
-     */
-    public function getHeaderLogoUrl($pathOnly = false)
-    {
-        $logo = new CustomLogo();
-        return $logo->getHeaderLogoUrl($pathOnly);
-    }
-
-    /**
-     * Returns the URL to application SVG Logo
-     *
-     * @ignore
-     * @param bool $pathOnly If true, returns path relative to doc root. Otherwise, returns a URL.
-     * @return string
-     */
-    public function getSVGLogoUrl($pathOnly = false)
```

Comment:
the first 2 were marked as deprecated so figured I remove all logo related api methods